soak/run_soak_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+PYTEST_LOG_LEVEL=$(echo "${PYTEST_LOG_LEVEL:-"info"}" | tr '[:upper:]' '[:lower:]')
```

Comment:
Maybe it's better to default `PYTEST_LOG_LEVEL` to "debug" ?

Reply:
i think info is good enough. These will just be test runs for >24 hrs. Having those many debug logs can be nuisance.
With info level, I believe all the info and detailed error logs are printed whenever pytest failure is there.

Why did you think debug will be a better choice?

Reply:
> With info level, I believe all the info and detailed error logs are printed whenever pytest failure is there.

If that's the case i agree that `info` will be better!

> Why did you think debug will be a better choice?

I just thought that the logs will not show `debug` logs if a controller fails